src/SDK/Language/Android.php

```diff
@@ -318,79 +348,13 @@ public function getFiles(): array
             ],
             [
                 'scope'         => 'copy',
-                'destination'   => '/example/.gitignore',
```

Comment:
are we removing these example files?

Reply:
Just for the Java specific example, they are only used for dev testing while adding new features, and I never use the Java one